extensions.json

```
@@ -71,6 +71,12 @@
       "id": "alonski.prettier-for-handlebars-vscode",
       "repository": "https://github.com/ember-tooling/prettier-for-handlebars-vscode"
     },
+    {
+      "id": "AmazonWebServices.aws-toolkit-vscode",
+      "repository": "https://github.com/aws/aws-toolkit-vscode",
+      "version": "1.15.0",
+      "checkout": "v1.15.0"
```

I see that this project even provides official `.vsix` releases. Whenever that's the case, we prefer downloading & republishing official releases directly (instead of trying to repackage it ourselves with custom checkout and/or prepublish steps).

```suggestion
      "download": "https://github.com/aws/aws-toolkit-vscode/releases/download/v1.15.0/aws-toolkit-vscode-1.15.0.vsix",
      "version": "1.15.0"
```